nilearn/_utils/numpy_conversions.py

```diff
@@ -124,3 +126,46 @@ def as_ndarray(arr, copy=False, dtype=None, order='K'):
         raise ValueError("Type not handled: %s" % arr.__class__)
 
     return ret
+
+
+def csv_to_array(csv_path, delimiters=' \t,;', **kwargs):
+    """Read a CSV file by trying to guess its delimiter
+
+    Parameters
+    ----------
+
+    csv_path: string
+        Path of the CSV file to load.
+
+    delimiters: string
+        String containing the possible delimiters for the CSV file.
```

Comment:
say that each character relates to one of the possible dilimiters...other wise the "syntax" of the delimiter argument might remain obscure to some. Especially when wondering whether '\t' counts as one or not...